### PR TITLE
Delete and rewrite Dockerfile & docker-compose

### PR DESCRIPTION
**Pull request by alefvanoon**

### CODE REVIEW

**Comment by zyachel** [comment]
Does alpine does the packaging faster than debian-based distros?

**Comment by zyachel** [comment]
Nevermind, I see what's wrong. Apparently, github doesn't notifiy in closed commits

**Comment by zyachel** [comment]
@alefvanoon, can you rebase?

File: docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ services:
 
   quetre:
     container_name: quetre
-    image: quay.io/pussthecatorg/quetre:latest
```
Comment (Odyssey346) [inline docker-compose.yml]: 👀

**Pull request closed without merging**